gui/wxpython/xml/toolboxes.xml

```diff
@@ -1076,6 +1076,10 @@
       <module-item name="r.fillnulls">
         <label>Fill NULL cells</label>
       </module-item>
+      <separator/>
+      <module-item name="r.fill.stats">
+        <label>Fill NULL cells with IDW</label>
```

Comment:
Would 'using IDW' sound better?

Otherwise looks good! Thank you

Reply:
Yes, you're right.